content/Tjenester/fiksprotokoll/brukerveiledning/huskeliste_for_opprette_protokoll.md

```diff
@@ -43,7 +48,16 @@ Når *"system*" er opprettet skal integrasjonsid og integrasjonspassord som man
 Hvis leverandør skal bruke API for resten av oppsett er man ferdig, hvis ikke se videre i huskelisten
 
 ### 5. Gyldig generert public key
-Når man setter opp en *"protokoll-konto"* i forvaltningssidene  må leverandøren ha gjort klar en public-key som man har generert på forhånd i korrekt format. Dette må personen fra kundeorganisasjon ha fått slik at man kan laste det opp i forvaltningssidene. 
+Når man setter opp en *"protokoll-konto"* i forvaltningssidene  må leverandøren ha gjort klar en public-key som man har generert på forhånd i korrekt format. 
+Dette må personen fra kundeorganisasjon ha fått slik at man kan laste det opp i forvaltningssidene. 
+Vi anbefaler at leverandøren deler filen slik at den kan lastes ned.
+
+**Husk:** Ikke alle e-post kontoer godtar filformatene for public-key og deling via e-post anbefales dermed ikke.
+
 
 ### 6. Valg av protokoll og part
 Leverandør(ene) har klart hvilken protokoll og hvilken *"part*" sin side skal være i kommunikasjonen
+
+### 7. Gi tilgang
```

Comment:
Forslag til tekst under. Bør man også si noe om at man ikke bare gir konto-til-konto tilgang, men også systemtilgang?

Man må huske å gi tilganger mellom kontoene. 

Det enkleste er å gjøre det i kontekst av "tjener" kontoen.
Derfra søker man opp og gir tilgang til "klient" kontoen, slik at denne kan sende meldinger til en selv.

Alternativt kan man gi tilgang i kontekst av "klient" systemet.
Derfra søker man opp "tjener" kontoen man ønsker å sende meldinger til og ber om å få tilgang. 
I etterkant må man så gå inn på "tjener" kontoen og bekrefte tilgangsforespørselen.

Reply:
Jeg er også usikke på teksten her. "Tjener" og "klient" er ikke nødvendigvis helt kjente uttrykk. Det er jo nesten så vi kunne hatt noen skjermbilder her og noen flere steder.
Med tekst ville jeg kanskje vurdert noe som
"... Det enkleste er å gjøre dette for kontoen som skal motta meldinger. Der kan man ved å gå til system-oversikt via "søk etter systemer"-fanen finne systemet i listen man ønsker å sende fra og gi tilgang.
Man kan alternativt gå inn på systemet man ønsker å sende fra og søke om tilgang til kontoen man ønsker å sende til.
Dette gjøres via "søk etter kontoer"-fanen for systemet.
Deretter må man gå til kontoen som skal motta meldinger, og inn på fanen "kontoer". Her vil forespørselen om tilgang dukke opp og man kan godta denne.

Jeg merket meg da jeg skrev dette at det er ingen måte å få dette til å virke helt enkelt... Men å bruke "klient" og "tjener" tror jeg blir for generelt for mange.